deployment/articles-dxp/01-saml/01-intro.markdown

```diff
@@ -0,0 +1,271 @@
+---
+header-id: authenticating-using-saml
+---
+
+# Authenticating Using SAML
+
+The SAML (Security Assertion Markup Language) adapter provides Single Sign On
+(SSO) and Single Log Off (SLO) in your deployment. Each @product@ instance
+serves as either the Service Provider (SP) or the Identity Provider (IdP). An
+identity provider is a trusted provider that provides single sign-on for users
+to access other websites. A service provider is a website that hosts
+applications and grants access only to identified users with proper credentials.
+
+| **Note:** A single @product@ instance is *either* the SP or the IdP in your SSO
+| setup; it can't be both. You can, however, use separate instances for both
+| purposes (for example, one instance is the SP and another is the IdP).
+
+Below is background on how SAML works. To jump right to its configuration, see
+the articles on 
+[Setting Up SAML as an Identity Provider](/docs/7-2/deploy/-/knowledge_base/deploy/setting-up-liferay-as-a-saml-identity-provider)
+or 
+[Setting Up SAML as a Service Provider](/docs/7-2/deploy/-/knowledge_base/deploy/setting-up-liferay-as-a-saml-service-provider) 
+for instructions on using the 
+[SAML adapter](https://web.liferay.com/marketplace/-/mp/application/15188711). Use
+the instructions to make the conceptual magic from this article come to life!
+
+| **Note:** If you're migrating from a Liferay SAML adapter prior to version
+| 3.1.0, your properties are automatically migrated to settings. Please see the
+| [Configuring SAML](/discover/deployment/-/knowledge_base/7-1/configuring-saml)
```

Comment:
Hey @sez11a 

Do we intentionally link back to the 7.1 version of the that doc?

Reply:
Nope. I probably didn't know what the new URLs would be when I wrote it. Now I do, and I'll fix them.